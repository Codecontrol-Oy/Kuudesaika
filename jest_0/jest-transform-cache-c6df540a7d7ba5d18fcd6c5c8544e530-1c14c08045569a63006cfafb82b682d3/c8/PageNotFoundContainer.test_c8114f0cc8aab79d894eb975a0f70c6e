'use strict';var _react = require('react');var _react2 = _interopRequireDefault(_react);
var _PageNotFoundContainer = require('./PageNotFoundContainer');var _PageNotFoundContainer2 = _interopRequireDefault(_PageNotFoundContainer);function _interopRequireDefault(obj) {return obj && obj.__esModule ? obj : { default: obj };}

describe("<PageNotFoundContainer />", function () {

    it("Matches snapshot", function () {
        var pageNotFoundContainer = shallow(
        _react2.default.createElement(_PageNotFoundContainer2.default, null));

        expect(pageNotFoundContainer).toMatchSnapshot();
    });
});