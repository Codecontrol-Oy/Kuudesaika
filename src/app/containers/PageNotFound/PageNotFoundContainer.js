import React from 'react';

export default class PageNotFoundContainer extends React.Component {

    constructor (props) {
        super(props);
    }

    render () {
      return (
        <div className="col-xs-12 col-sm-12 col-md-12 col-lg-12">
            <span>Page not found!</span>
        </div>
      );}

}
