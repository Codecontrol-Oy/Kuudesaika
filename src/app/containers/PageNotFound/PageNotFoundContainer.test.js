import React from 'react';
import PageNotFoundContainer from './PageNotFoundContainer';

describe("<PageNotFoundContainer />", () => {

    it("Matches snapshot", () => {
        const pageNotFoundContainer = shallow(
            <PageNotFoundContainer />
        );
        expect(pageNotFoundContainer).toMatchSnapshot();
    });
}) 
