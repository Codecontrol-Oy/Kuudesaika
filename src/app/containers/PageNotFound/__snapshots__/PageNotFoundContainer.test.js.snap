// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PageNotFoundContainer /> Matches snapshot 1`] = `
<div
  className="col-xs-12 col-sm-12 col-md-12 col-lg-12"
>
  <span>
    Page not found!
  </span>
</div>
`;
